keyboards/ergodox_ez/rules.mk

```diff
@@ -30,7 +30,6 @@ UNICODE_ENABLE   = yes  # Unicode
 SWAP_HANDS_ENABLE= yes  # Allow swapping hands of keyboard
 SLEEP_LED_ENABLE = no
 API_SYSEX_ENABLE = no
-RGBLIGHT_ENABLE = yes
 
```

Comment:
Cant make the suggestion on a nice line, but to maintain the existing behaviour, you would want something like
`DEFAULT_FOLDER = ergodox_ez/shine`

Reply:
This one is complicated, and I think it may be best to leave this, as is. 
Specifically, there are 3 different SKUs for the ErgoDox EZ, *and* this supports the OG Ergodox too (which doesn't support rgblight, at all).  

Having the rgb light enabled when it isn't actually present doesn't adversely effect the board (at least, not in any noticeable way).   And anyone dumping the files from the ErgoDox EZ Configurator will have this enabled in their rules.mk, if it's a shine model.